doc/user-guide/custom-domains.md

```diff
@@ -21,49 +21,73 @@ If you would like a custom `ploomberapp.io` subdomain, like `my-custom-app.ploom
 
 ### Adding a custom subdomain
 
-To deploy an app at `subdomain.yourdomain.com`, you first need to ensure it's deployed and available (the **VIEW APPLICATION** button should be enabled)
+To deploy an app at `subdomain.yourdomain.com`, you first need to ensure the main domain you want to link to (e.g., `yourdomain.com`) is registered properly. 
```

Comment:
Currently the sequence is:
Adding a custom domain/subdomain
  - Adding a custom subdomain
     - Adding a custom domain
     
Can we change the sequence to include custom domain first?
Adding a custom domain/subdomain
  - Adding a custom domain
  - Adding a custom subdomain

It's easier to understand subdomains after the custom domain section.

Reply:
[fixed](https://github.com/ploomber/doc/pull/168/commits/728ed38ec608321ebfcd672a61e45da7d6414843)